.github/workflows/validate-migration.yaml

```diff
@@ -57,7 +115,47 @@ jobs:
       - name: "Install c++ compilation dependencies"
         run: |
           sudo apt-get update
-          sudo apt-get install git make cmake gcc g++ clang --yes # p4-fusion dependencies
+          sudo apt-get install --yes \
+            git \
+            make \
+            cmake \
+            gcc \
+            g++ \
+            clang \
+            libc6-dbg \
+            libc++-dev \
+            libc++abi-dev \
+            llvm \
```

Comment:
Mention that `llvm` is being installed for `llvm-symbolizer` which is needed for symbolicating stack traces.